lib/cli/build.rs

```diff
@@ -16,6 +16,8 @@ pub fn main() {
             "cargo:rustc-env=WASMER_BUILD_GIT_HASH_SHORT={}",
             &git_hash[..5]
         );
+    } else {
+        println!("cargo:rustc-env=WASMER_BUILD_GIT_HASH_SHORT=??????");
```

Comment:
This is the only thing I'm unsure about - for some reason the git hash is not available on -musl, not sure if it's 100% necessary, this is more of a quick hack